libcloud/storage/drivers/s3.py

```diff
@@ -1435,6 +1395,17 @@ class S3EUWest2StorageDriver(S3StorageDriver):
     region_name = "eu-west-2"
 
 
+class S3EUWest3Connection(S3SignatureV4Connection):
```

Comment:
Those two classes are actually not needed anymore - this was an old approach (class per region) which is not used anymore and has been replaced via `region` class constructor argument.

Can you please remove them? Thanks.

Reply:
I removed them :slightly_smiling_face: